runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -1217,6 +1217,17 @@ class INTERPRETER_CLASS
 				}
 			}
 		}
+#if JAVA_SPEC_VERSION >= 24
+		if (rc == (UDATA)obj) {
+
+			J9VMContinuation *continuation = _currentThread->currentContinuation;
+			if (NULL != continuation) {
+				/* Do not need to recored whether if contend monitor enter event is triggered after entered the monitor */
```

Comment:
minor formatting nit: remove empty line and fix comment (typo and period)
```suggestion
			J9VMContinuation *continuation = _currentThread->currentContinuation;
			if (NULL != continuation) {
				/* Do not need to record whether if contended monitor enter event is triggered after entering the monitor. */
```